Checkers/Checkers.cs

```diff
@@ -8,79 +8,435 @@ class Program
+                    inBetweenRow = check.coords.y + 1;
+                }
+                //otherwise change in y is negative and the in between y will be starting y -1
+                else
+                {
+                    inBetweenRow = check.coords.y - 1;
+                }
+            }
+
+            //otherwise change in x is negative, the inbetween x will be starting x - 1
+            else
+            {
+                inBetweemColumn = check.coords.x - 1;
+                //if the change in y is positive, the inbetween y will be starting y + 1
+                if ((newRow - check.coords.y) > 0)
+                {
+                    inBetweenRow = check.coords.y + 1;
+                }
+                //otherwise change in y is negative and the in between y will be starting y -1
+                else
+                {
+                    inBetweenRow = check.coords.y - 1;
+                }
+            }
+            //use the in between row and column to select the in between checker
+            inBetweenCheck = SelectChecker(inBetweenRow, inBetweemColumn);
+
+            //if there's a checker there and it's the opposite color of the starting checker, then remove it and move the starting checker to the ending coords
+            if (inBetweenCheck != null)
+            {
+                if (inBetweenCheck.color != check.color)
+                {
+                    RemoveChecker(inBetweenRow, inBetweemColumn);
+                    Move(check, newRow, newColumn);
+                }
+            }
+            else
+            {
+                throw new Exception ("that is not a valid jump");
+            }
         }
         
+        //removeChecker function, takes in a row and column, and removes the checker at this position from the list, and updates the physical grid to a blank space at that position
         public void RemoveChecker(int row, int column)
         {
-            // Your code here
-            return;
+            Checkers.Remove(SelectChecker(row, column));
+            Grid[row][column] = " ";
         }
         
+        //check for win bool, returns true if the only checkers left are all of one color
         public bool CheckForWin()
         {
-            return Checkers.All(x => x.Color == "white") || !Checkers.Exists(x => x.Color == "white");
+            return Checkers.All(x => x.color == Color.white) || !Checkers.Exists(x => x.color == Color.white);
         }
     }
 
+
     class Game
     {
         public Game()
         {
-            // Your code here
+            Board board = new Board(); 
+            board.DrawBoard();
+
+            string playerTurn = "White";
+            bool swap = true;
+
+            while(board.CheckForWin() == false)
+            {
+                if (swap)
+                {
+                    playerTurn = (playerTurn == "White" ? "Black" : "White");   
+                }
+
+                Console.WriteLine("Player Turn: " + playerTurn);
```

Comment:
You don't handle bad user input that is not a number.
You could put this code in a try catch to handle bad user input